docs/modules/ROOT/pages/clc-shell.adoc

```diff
@@ -0,0 +1,53 @@
+= clc shell
```

Comment:
description: The interactive shell of the Hazelcast CLC allows you to run SQL queries and CLC commands.

Also, it would be good to move the Usage: to near the top of the page.